### PR TITLE
Improve instruction for deploy Kyuubi on K8s

### PR DESCRIPTION
**Pull request by pan3793**
# :mag: Description
## Issue References 🔗

Historically, we provided a few standalone YAML files to allow users to deploy Kyuubi on K8s, and later a Helm Chart was introduced, which is a simple and mature way for production use cases.

## Describe Your Solution 🔧

Remove the legacy standalone YAML files way and update docs to guide user to use Helm chart to deploy Kyuubi on K8s.

## Types of changes :bookmark:

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Test Plan 🧪

Review

---

# Checklist 📝

- [x] This patch was not authored or co-authored using [Generative Tooling](https://www.apache.org/legal/generative-tooling.html)

**Be nice. Be informative.**

### CODE REVIEW

File: charts/kyuubi/values.yaml [hunk 1]
```diff
@@ -45,7 +45,7 @@ updateStrategy:
 image:
   repository: apache/kyuubi
   pullPolicy: IfNotPresent
-  tag: ~
+  tag: 1.8.1-spark
```
Comment (pan3793) [inline charts/kyuubi/values.yaml:48]: use an image that contains Spark binary as the default value.

File: docker/Dockerfile [hunk 2]
```diff
@@ -15,14 +15,12 @@
...
 
-# Usage:
-#   1. use ./build/dist to make binary distributions of Kyuubi or download a release
-#   2. Untar it and run the docker command below
-#      docker build -f docker/Dockerfile -t repository/kyuubi:tagname .
-#   Options:
-#     -f this docker file
-#     -t the target repo and tag name
-#     more options can be found with -h
+# It's recommended to use ./bin/docker-image-tool.sh to build the image rather than the vanilla docker command.
```
Comment (pan3793) [inline docker/Dockerfile:18]: cc @zwangsheng, am I right?

File: charts/kyuubi/values.yaml (hunk 1 above)
Comment (dnskr) [reply charts/kyuubi/values.yaml:48]: I would prefer to leave `~` for now to use `{{ .Chart.AppVersion }}` as image tag value by default.
The change makes change, however it's not enough to deploy Kyuubi Helm chart with Spark support.
Let me create `Minimal Helm chart configuration to run Kyuubi with Spark` issue to discuss all problems, limitations and options there.

File: docker/Dockerfile (hunk 2 above)
Comment (zwangsheng) [reply docker/Dockerfile:18]: Yes, although we may need to optimize this script for now, we still recommend that users use script builds to reduce the complexity of building docker images.

**Review by zwangsheng — APPROVED** [review]
LGTM

**Comment by zwangsheng** [comment]
BYW, should we mentions update `docker-image-tool.sh` in release guide?

File: charts/kyuubi/values.yaml (hunk 1 above)
Comment (pan3793) [reply charts/kyuubi/values.yaml]: reverted

**Comment by pan3793** [comment]
Thanks, merged to master

**Pull request closed without merging**